charts/m4d/Chart.yaml

```diff
@@ -5,12 +5,10 @@ apiVersion: v2
 name: m4d
 description: Mesh for Data Helm Chart
 type: application
-
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 0.1.0
-
+version: latest
 # This is the version number of the application being deployed. This version number should be
 # incremented each time you make changes to the application.
-appVersion: 0.1.0
+appVersion: latest
```

Comment:
a minor comment: this means that "--set global.tag=latest" is not needed anymore in the quickstart guide, right?

Reply:
right but I'm keeping it for being future proof if we start using the x.y.z-SNAPSHOT approach